docs/user/aws/install_upi.md

```diff
@@ -32,6 +32,21 @@ $ tree
 └── worker.ign
 ```
 
+### Extract Infrastructure ID from Ignition Metadata
+
+Many of the operators and functions within OpenShift rely on tagging AWS resources. By default, Ignition
+generates a unique cluster identifier comprised of the cluster name specified during the invocation of the installer
+and a short string known internally as the infrastructure ID. These values are seeded in the initial manifests within
+the Ignition configuration. To use the output of the default, generated 
+`ignition-configs` extracting the internal infrastructure ID is necessary.
+
+An example of a way to get this is below (`vw9j6` is the infrastructure ID): 
```

Comment:
nit: I'd drop the parenthetical.  I think the whole `openshift-vw9j6` is infrastructure ID, but the docs don't have to be opinionated one way or another.  But we can clean this up in follow-up work, no need to push another round of CI for this.

Reply:
Resolved.